xbmc/addons/Addon.cpp

```diff
@@ -412,6 +413,8 @@ void CAddon::BuildLibName(const cp_extension_t *extension)
       case ADDON_SERVICE:
       case ADDON_REPOSITORY:
       case ADDON_CONTEXT_ITEM:
+      case ADDON_AUDIODECODER:
```

Comment:
3 years of constant rebasing, where things originating in my branch has been pushed to mainline in slightly altered forms...